hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/common/HoodieSparkEngineContext.java

```diff
@@ -62,8 +62,7 @@
  */
 @ThreadSafe
 public class HoodieSparkEngineContext extends HoodieEngineContext {
-
-  private static final Logger LOG = LoggerFactory.getLogger(HoodieSparkEngineContext.class);
+  
   private final JavaSparkContext javaSparkContext;
   private final SQLContext sqlContext;
   private final Map<HoodieDataCacheKey, List<Integer>> cachedRddIds = new HashMap<>();
```

Comment:
Also needs to fix the unnecessary import.

Reply:
ok, done